packages/enketo-core/src/js/calculate.js

```diff
@@ -482,7 +482,7 @@ export default {
                         const preInitRelevance =
                             this.preInitRelevance.get(element);
 
-                        if (preInitRelevance != null) {
+                        if (preInitRelevance != null && !this.initialized ) {
```

Comment:
I believe the intent of the preInitRelevance cache is to be used only during initialization and not afterwards.